CMakeLists.txt

```
@@ -28,16 +28,27 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # throughout source files.
 set(GPU_DEVICE_NONE 0)
 set(GPU_DEVICE_CUDA 1)
+set(GPU_DEVICE_HIP  1)  # NOTE: HIP is treated as CUDA in this project
```

For consistency and clarity (sorry, this was my addition)
```suggestion
set(GPU_DEVICE_HIP  1)  # NOTE: HIP is treated as CUDA in FTorch
```